charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml

```diff
@@ -233,6 +236,10 @@ spec:
         hostPath:
           path: /sys
           type: Directory
+      - name: host-root
```

Comment:
Let's make it configurable and do not share host root by default.

Reply:
Done